docs/loading/Loading_data_template.md

```diff
@@ -0,0 +1,348 @@
+
+### Data flow
+
+***Note: Processes that involve multiple components or steps may be easier to understand with a diagram. This example includes a diagram that helps describe the steps that happen when a user chooses the Broker Load option.***
+
+![Workflow of Broker Load](../assets/broker_load_how-to-work_en.png)
+
+1. The user creates a load job
+2. The frontend (FE) creates a query plan and distributes the plan to the backend nodes (BE)
+3. The backend (BE) nodes pull the data from the source and load the data into StarRocks
+
+### Typical example
+
+Create a table, start a load process that pulls a Parquet file from S3, and verify the progress and success of the data loading.
+
+> Note
+>
+> The examples use a sample dataset in Parquet format, if you want to load a CSV or ORC file, that information is linked at the bottom of this page.
+
+#### Create a table
+
+Create a database for your table:
+
+```SQL
+CREATE DATABASE IF NOT EXISTS project;
+USE project;
+```
+
+Create a table. This schema matches a sample dataset that you can load from a StarRocks S3 bucket.
+
+```SQL
+DROP TABLE IF EXISTS user_behavior;
+
+CREATE TABLE `user_behavior` (
+    `UserID` int(11),
+    `ItemID` int(11),
+    `CategoryID` int(11),
+    `BehaviorType` varchar(65533),
+    `Timestamp` datetime
+) ENGINE=OLAP 
+DUPLICATE KEY(`UserID`)
+DISTRIBUTED BY HASH(`UserID`)
+PROPERTIES (
+    "replication_num" = "1"
+);
+```
+
+#### Gather connection details
+
+> Note
+>
+> The examples use IAM-based authentication. Other authentication methods are available and linked at the bottom of this page.
+
+Loading data from S3 requires having the:
+
+- S3 bucket
+- S3 object keys (object names) if accessing a specific object in the bucket
+- S3 region
+- Access key and secret
+
+#### Start a Broker Load
+
+This `LOAD` job has four main sections:
+
+- `LABEL`: A string used when querying the state of a `LOAD` job.
+- `LOAD` declaration: The source URI, destination table, and the source data format.
+- `BROKER`: The connection details for the source.
+- `PROPERTIES`: Timeout value and any other properties to apply to this job.
+
+> Tip
+>
+> The dataset used in these examples is available in a StarRocks S3 bucket. Any valid `aws.s3.access_key` and `aws.s3.secret_key` can be used, as the object is readable by any AWS authenticated user. Substitute your credentials for `AAA` and `BBB` in the commands below.
+
+```SQL
+LOAD LABEL user_behavior
+(
+    DATA INFILE("s3://starrocks-datasets/user_behavior_sample_data.parquet")
+    INTO TABLE user_behavior
+    FORMAT AS "parquet"
+ )
+ WITH BROKER
+ (
+    "aws.s3.enable_ssl" = "true",
+    "aws.s3.use_instance_profile" = "false",
+    "aws.s3.region" = "us-west-1",
+    "aws.s3.access_key" = "AAAAAAAAAAAAAAAAAAAA",
+    "aws.s3.secret_key" = "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
+ )
+PROPERTIES
+(
+    "timeout" = "72000"
+);
+```
+
+#### Check progress
+
```

Comment:
should this sec include `SELECT * from user_behavior_inferred LIMIT 10;` to query data in the table after loading.